Source/JavaScriptCore/runtime/VM.h

```diff
@@ -302,7 +302,7 @@ class VM : public ThreadSafeRefCounted<VM>, public DoublyLinkedListNode<VM> {
     enum VMType { Default, APIContextGroup, APIShared };
 
     struct ClientData {
-        JS_EXPORT_PRIVATE virtual ~ClientData() = 0;
+        JS_EXPORT_PRIVATE virtual ~ClientData() { };
```

Comment:
Perhaps this is the best workaround possible, but this does change the interface - subclasses no longer have to override the destructor.

Reply:
I think this is fine.